CONTRIBUTING.md

```diff
@@ -23,10 +23,9 @@ you can fix this by specifying appropriate [**CSS selectors**](https://developer
 **[dynamic-theme-fixes.config](https://github.com/darkreader/darkreader/blob/master/src/config/dynamic-theme-fixes.config)**
 (for Dynamic Theme mode)
 or **[inversion-fixes.config](https://github.com/darkreader/darkreader/blob/master/src/config/inversion-fixes.config)**
-(for Filter and Filter+ modes)
-*(please, preserve alphabetical order by URL, use short selectors, and preserve code style)*.
+(for Filter and Filter+ modes). **Please, preserve alphabetical order by URL, use short selectors, and preserve code style).**
```

Comment:
No,
This line applies to both inversion and dynamic-theme-fixes. This looks like it just applies for the inversion-fixes.